pyproject.toml

```diff
@@ -42,7 +42,7 @@ msgpack = "^1.0.0"
 neo4j = "^4.1.1"
 pylnk3 = "^0.4.2"
 pypsrp = "^0.7.0"
-paramiko = "^2.7.2"
+paramiko = "=3.3.1"
```

Comment:
What's needed in the new version?

Reply:
New version has better performance and lots of bugs fixed

Reply:
Is there a reason to pin the version to 3.3.1? Why not `^3.3.1`

Reply:
I'm not good at using poetry stuff XD, it's better to use `^3.3.1`?

Reply:
Haha no problem. With `=` you pin the installed version to exactly the specified version. Using `^` specifies that installed version must atleast match the specified version but allows updates with `poetry update` to higher versions except a major version bump. (The exact version that is installed then is specified in the lock file)
So probably we should use `^`.

See: https://python-poetry.org/docs/dependency-specification/

Reply:
Ha, I see, thanks you!